messenger.rst

```diff
@@ -2836,6 +2836,11 @@ and a different instance will be created per bus.
             $bus->middleware()->id('App\Middleware\AnotherMiddleware');
         };
 
+.. tip::
+
+    If you have installed the Maker bundle, you can use the ``make:messenger-middleware`` command
```

Comment:
```suggestion
    If you have installed the MakerBundle, you can use the ``make:messenger-middleware`` command
```
Cosmetic but I have the impression to see it written this way most of the time in documentation
And we may can add link to bundle documentation too

Reply:
I didn't find a way to link to the bundle documentation (see first commit). So I checked some other pages and often there was no link to the bundles pages.